_studio/mfx_lib/vpp/src/mfx_vpp_sw_core.cpp

```diff
@@ -426,8 +426,27 @@ mfxStatus VideoVPPBase::QueryIOSurf(VideoCORE* core, mfxVideoParam *par, mfxFram
 
         mfxSts = CheckIOPattern_AndSetIOMemTypes(par->IOPattern, &(request[VPP_IN].Type), &(request[VPP_OUT].Type), bSWLib);
         MFX_CHECK_STS(mfxSts);
-        return (bSWLib)? MFX_ERR_UNSUPPORTED : MFX_ERR_NONE;
+        return (bSWLib)? MFX_WRN_PARTIAL_ACCELERATION : MFX_ERR_NONE;
     }
+    else
+    {
+        // since MSDK_3.0 asyncDepth is mandatory
```

Comment:
Looks like we should replace this whole else block with `return MFX_ERR_UNSUPORTED;`